### PR TITLE
HEEDLS-383 Change spacing around buttons and add table caption

### PR DESCRIPTION
**Pull request by AlexJacksonDS**
Bug fixes for 383. Remove buttons in the answer table are now vertically aligned, and I've added the 'Configured answers:' caption to the table for the Edit screen.

image

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/_RegistrationPromptAnswerTable.cshtml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ﻿@using DigitalLearningSolutions.Web.Controllers.TrackingSystem.CentreConfiguration
-@model List<string>
+@model (List<string>, bool)
```
Comment (stellake) [inline DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/_RegistrationPromptAnswerTable.cshtml:2]: Could we create a new model for this to make things a bit neater now that we have different fields too?

Comment (AlexJacksonDS) [reply DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/_RegistrationPromptAnswerTable.cshtml]: Added a new property to the base model and just passing the whole thing in instead. Seems cleaner than creating a whole new model that I would have to populate somewhere

File: DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/_RegistrationPromptAnswerTable.cshtml [hunk 2]
```diff
@@ -13,14 +17,13 @@
   </tr>
   </thead>
   <tbody class="nhsuk-table__body">
-  @foreach (var answer in Model!)
-  {
+  @foreach (var answer in Model.Options!) {
```
Comment (stellake) [inline DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/_RegistrationPromptAnswerTable.cshtml:20]: Is Model.Options nullable?

File: DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/_RegistrationPromptAnswerTable.cshtml (hunk 1 above)
Comment (stellake) [reply DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/_RegistrationPromptAnswerTable.cshtml]: That's neat! I wonder if we could call the boolean "IncludeAnswersTableCaption" or something similar to make it clear what it does in the UI?

**Review by stellake — APPROVED** [review]
Looks great! 👍

**Pull request merged**